10-inference.Rmd

```diff
@@ -454,7 +442,7 @@ in a more reliable point estimate of the population parameter.
 ### Summary
 1. A *point estimate* is a single value computed using a sample from a population (e.g. a mean or proportion)
 2. The *sampling distribution* of an estimate is the distribution of the estimate for all possible samples of a fixed size from the same population.
-3. The sample means and proportions calculated from samples are centered around the population mean and proportion, respectively.
+3. The sample means and proportions calculated from samples are centred around the population mean and proportion, respectively.
 4. The spread of the sampling distribution is related to the sample size. As the sample size increases, the spread of the sampling distribution decreases. 
 5. The shape of the sampling distribution is usually bell-shaped with one peak and centred at the population mean or proportion.
 
```

Comment:
The paragraph below (Why all this emphasis on...?) needs heavy editing to flow better. Lots of internal redundancy, and redundancy with the text at the start of bootstrapping.

Since it's basically a motivation for bootstrapping, I would strongly suggest that we remove it from this section and merge it with the text at the beginning of 11.5.

Reply:
Removed and merged with the section below! And checked for redundancy